CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to Rust's notion of
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Removed
+- `halo2::arithmetic::BatchInvert` (use `ff::BatchInvert` instead).
```

Comment:
Sigh, I thought I'd done this before the first beta (which depends on `ff 0.11` which includes the upstreamed trait).